resources/todomvc/architecture-examples/preact/src/todo/components/input.jsx

```diff
@@ -18,6 +18,16 @@ const hasValidMin = (value, min) => {
 };
 
 export function Input({ onSubmit, placeholder, label, defaultValue, onBlur }) {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
```

Comment:
can you please add a comment explaining what this `useEffect` does? For example `Focus and put the cursor at the end of the input when the input appears` (if I understand this well).

Reply:
will add!

Reply:
BTW I see that hte react PR simply uses `autoFocus` for this, doesn't that work here?

Reply:
that works for the main input in the header, but doesn't work for the input in each item when you try to edit the todo

Reply:
That's weird :-) can you please add this explanation to the comment?